frontend/pages/sacfairguide.tsx

```diff
@@ -0,0 +1,285 @@
+import s from 'styled-components'
+
+import { Contact, Container, Metadata, Title } from '../components/common'
+import { SNOW } from '../constants/colors'
+import { M4 } from '../constants/measurements'
+import renderPage from '../renderPage'
+
+type QuestionProps = React.PropsWithChildren<{
+  title: string
+}>
+
+const Subheading = s.div`
+  margin-top: 1.5rem;
+  font-size: ${M4};
+  font-weight: bold;
+  margin-bottom: 0.5rem;
+`
+
+const UnorderedList = s.ul`
+  list-style-type: disc;
+  margin-left: 2rem;
+  margin-top: 0.5rem;
+`
+
+const SACGuide = () => (
+  <Container background={SNOW}>
+    <Metadata title="SAC Fair Guide" />
+    <Title style={{ paddingTop: '2.5vw', paddingBottom: '2rem' }}>
+      Virtual SAC Fair Guide
+    </Title>
+    <p>
+      Hi there! Welcome to <strong>Penn Clubs</strong>, the University of
+      Pennsylvania's official registry for student organizations on campus. The
+      purpose of this guide is to walk you through the many different features
+      that Penn Clubs can offer that will make your experience recruiting new
+      members as easy as possible.
+    </p>
+
+    <Subheading>REQUIRED: Editing Your SAC Fair "Booth"</Subheading>
+
+    <p>
+      You will need to add a Zoom link to your club's SAC Fair event. Once you
+      navigate to your club, click the green "Manage Club" button on the
+      top-right corner of the page. If you do not see this button, it means you
+      are not an officer of the club. Either ask an officer listed on your club
+      to add you to the site through the "Membership" tab of the "Manage Club"
+      page, or email <Contact /> with your Penn email and club name to be added
+      as an officer.
+    </p>
+    <img
+      style={{
+        maxHeight: 400,
+        verticalAlign: 'middle',
+        margin: 10,
+        padding: 10,
+      }}
+      src={'static/img/sacfair_screencaps/1.png'}
+      alt={'Manage Club Button'}
+      title={'Manage Club Button'}
+    />
+    <p>Then, click the "Events" tab:</p>
+    <img
+      style={{
+        maxHeight: 400,
+        verticalAlign: 'middle',
+        margin: 10,
+        padding: 10,
+      }}
+      src={'static/img/sacfair_screencaps/2.png'}
+      alt={'Events Tab'}
+      title={'Events Tab'}
+    />
+
+    <p>
+      There will be an existing event with your designated date & time titled
+      "SAC Fair Info Session". Click "Edit".
+    </p>
+    <img
+      style={{
+        maxHeight: 400,
+        verticalAlign: 'middle',
+        margin: 10,
+        padding: 10,
+      }}
+      src={'static/img/sacfair_screencaps/3.png'}
+      alt={'Edit SAC Fair Event'}
+      title={'Edit SAC Fair Event'}
+    />
+    <p>
+      Edit the club details: you <strong>MUST</strong> provide a Zoom link
+      (preferably from your school Zoom account) where prospective students can
```

Comment:
Maybe talk about why school Zoom account, and the 40 minute meeting limit for free accounts.

Reply:
Ah, I see that this is mentioned below.